atlasaction/action.go

```diff
@@ -0,0 +1,36 @@
+package atlasaction
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"strconv"
+
+	"ariga.io/atlas-go-sdk/atlasexec"
+	"github.com/sethvargo/go-githubactions"
+)
+
+// MigrateApply runs the GitHub Action for "ariga/atlas-action/migrate/apply".
+func MigrateApply(ctx context.Context, client *atlasexec.Client, act *githubactions.Action) error {
+	params := &atlasexec.MigrateApplyParams{
+		URL:             act.GetInput("url"),
+		DirURL:          act.GetInput("dir"),
+		TxMode:          act.GetInput("tx-mode"),  // Hidden param.
+		BaselineVersion: act.GetInput("baseline"), // Hidden param.
+	}
```

Comment:
This is not going to work when running the action as a Go binary using composite action unless you somehow load the environment variables 

see my comment here: https://github.com/ariga/atlas-push-action/blob/cb452771d310af850d1eb64abd9b25de72003cd5/internal/atlasaction/actions.go#L110

Reply:
I have some ideas for ways around the issue, don't worry about it for now

Reply:
Do share, I'm curious :)